lib/src/components/NxTag/NxTag.tsx

```diff
@@ -48,14 +48,12 @@ export const NxSelectableTag: FunctionComponent<SelectableProps> =
             <NxFontAwesomeIcon icon={isSelected ? faTimesCircle : faPlusCircle} className="nx-tag__action" />;
 
       return (
-        <NxTag tabIndex={0}
-               role="switch"
```

Comment:
Based on its definition (https://www.w3.org/TR/wai-aria/#switch) I feel like this role might still be a good idea

Reply:
> Based on its definition (https://www.w3.org/TR/wai-aria/#switch) I feel like this role might still be a good idea

Yes: https://github.com/sonatype/sonatype-react-shared-components/pull/566/commits/90758fbf2cfb7327222c9f02b17e385c4620a728

Reply:
> Yes: 90758fb

The switch role requires the `aria-checked` attr according to the spec (though I'm not sure why it really needs to in this case)

Reply:
> > Yes: [90758fb](https://github.com/sonatype/sonatype-react-shared-components/commit/90758fbf2cfb7327222c9f02b17e385c4620a728)
> 
> The switch role requires the `aria-checked` attr according to the spec (though I'm not sure why it really needs to in this case)

Well it's got it now: https://github.com/sonatype/sonatype-react-shared-components/pull/566/commits/a0ec7fb731847f9020a08368f91254e1fe100fdb